bftengine/src/preprocessor/tests/preprocessor_test.cpp

```diff
@@ -370,13 +371,20 @@ void setUpConfiguration_7() {
 }
 
 void setUpCommunication() {
-  unordered_map<NodeNum, NodeInfo> nodes;
-
-  NodeInfo nodeInfo{"128.0.0.1", 4321, true};
-  nodes[0] = nodeInfo;
-
-  PlainUdpConfig configuration("128.0.0.1", 1234, 4096, nodes, replicaConfig.replicaId);
-  communicatorPtr.reset(CommFactory::create(configuration), [](ICommunication* c) {
+  auto logger = logging::getLogger("preprocessor_test");
```

Comment:
This is horrible, we need to add a ticket for isolating the ifdefs in a factory

Reply:
agree